assets/app/view/game/game_info.rb

```diff
@@ -147,62 +116,66 @@ def upcoming_trains
         rows = @depot.upcoming.group_by(&:name).map do |name, trains|
           train = trains.first
           discounts = train.discount&.group_by { |_k, v| v }&.map do |price, price_discounts|
-            price_discounts.map(&:first).join(',') + ' => ' + @game.format_currency(price)
+            price_discounts.map(&:first).join(', ') + ' ⮕ ' + @game.format_currency(price)
```

Comment:
same here

Reply:
> can you change "Operating Order" to be "Order"

Sure. I’ll forward any hate-mail from spreadsheets grognards to you ;)